### ISSUE TITLE
touchEnded() on mobile while dragging - TypeError: evt is undefined

### ISSUE THREAD

**Issue by eispfote** [processing/p5.js#1775]
Hey there! I'm having trouble with touchevents on mobile. 

I make this really simple example to show you my problem: https://codepen.io/Eispfote/pen/YNQyWX
When I tap I get "started", "moved" and "ended" but when I drag my finger I get only "started" and "moved". Everything is okay if I try it on desktop, but on mobile I get this error too:
(TypeError: d is undefined) or (TypeError: evt is undefined)

**Comment by tstefanich** [comment]
I think this is technically fixed on the github version already but not yet officially released. There is a difference in the version of p5 you download from the website (and in your sketch) versus the version on github. 

```
p5.prototype._updateNextMouseCoords = function(e) {
  if(this._curElement !== null) {
````

has been changed to 

```
p5.prototype._updateNextMouseCoords = function(e) {
if(this._curElement !== null && (!e.touches || e.touches.length>0)) {
```

When I download p5 from github and used your sketch I did not get any of the errors I got on your code pen example. 

Let me know if that resolves it for you. If so I think this issue can be closed.
